iframing things up

jhicken opened the pull request:
please have a look. lots needs to be done. Such as stateful stuff being pulled out of the iframe or ... styles being passed in. 

I would like to look at how storybook is doing it.

steos commented:
That devtool warning I mentioned in #12 seems to be a local problem, it works fine on another system. 

So I think this can be merged but as it is right now test cards and markdown cards are also rendered inside an iframe. I think we should do this only for the user components.

There also seems to be a slight issue with the markdown content being cut off due to being rendered inside the iframe (windows 7, chrome/ff/ie all the same)

![reactcards-markdown-iframe-cutoff](https://cloud.githubusercontent.com/assets/481021/15657156/a71bd852-26ae-11e6-87b6-7fe57b985d70.png)

But user components seem fine so I think we can just ignore this if we don't wrap the markdown cards.

jhicken commented:
Yeah I think that is an iframe-resizer issue.

So I would be happy to remove the wrapper from everything accept the user components. I don't know when i will have time but ill update the PR soon.

jhicken commented:
This may be an overloaded pr now however its adding a ton of goodness

I fixed the iframe stuffs. and fixed a bunch of indentation inconsistencies. I also included an .eslintrc and .editorconfig for code consistency.

@steos It might be easier for you to manually merge this. I included the lodash upgrade in it too.

steos commented:
cool, thanks a lot for all your work. I'll check it out as soon as I have some time (probably tomorrow).
